enos/enos-scenario-autopilot.hcl

```diff
@@ -42,7 +42,7 @@ scenario "autopilot" {
     module = "build_${matrix.artifact_source}"
 
     variables {
-      build_tags            = try(var.vault_local_build_tags, local.build_tags[matrix.edition])
+      build_tags            = var.vault_local_build_tags != null ? var.vault_local_build_tags : local.build_tags[matrix.edition]
```

Comment:
why replace `try` here 😕  ?

Reply:
Because it wasn't doing what we need it to do. The `vault_local_build_tags` are equal to `null` in this case, so try will return `null` because the expression is valid. `try()` only returns the second value if the first expression fails to evaluate, i.e. returns an error.